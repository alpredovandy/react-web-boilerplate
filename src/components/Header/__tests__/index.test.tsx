import React from "react";

import { render } from "@testing-library/react";

import Header from "../";
import type { HeaderProps } from "../types";

describe("Test components: Header", () => {
  let props: HeaderProps;
  const renderHeader = () => render(<Header {...props} />);

  it("Snapshot", () => {
    const { container } = renderHeader();

    expect(container).toMatchSnapshot();
  });
});
