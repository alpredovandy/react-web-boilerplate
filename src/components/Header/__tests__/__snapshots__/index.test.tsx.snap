// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test components: Header Snapshot 1`] = `
<div>
  <head>
    <title />
    <meta
      content="IE=edge,chrome=1"
      http-equiv="X-UA-Compatible"
    />
    <meta
      content="width=device-width, initial-scale=1.0, user-scalable=no"
      name="viewport"
    />
  </head>
</div>
`;
